README.md

```diff
@@ -196,6 +197,19 @@ The following table lists some of the usernames and their corresponding user typ
 ![Login page](./docs/figures/login.png "Login page")
 ![User logged in](./docs/figures/logged-in.png "Developer user logged in")
 
+### Adding Users
+
+To add a new user to the system, you must first define the user in the same format as the users in the
+`example-adding-users.json` file in the `etc/fixtures` directory.
+
+Then, you must run the following command from the root directory of the project:
+
+```bash
+docker-compose exec -T rosalution-db mongosh /tmp/fixtures/add_user.js example-adding-users.json
```

Comment:
When running the docker command, there appears to be an error. This is due to the mongosh command taking each argument passed and trying to execute each one.

First it accepts this and runs the script as it is asked and adds the users using the json as an argument.

`docker-compose exec -T rosalution-db mongosh **/tmp/fixtures/add_user.js** example-adding-users.json`

Then mongosh moves on to this file which it can't find and throws an error as seen below.

`docker-compose exec -T rosalution-db mongosh /tmp/fixtures/add_user.js **example-adding-users.json**`



<img width="542" alt="Screenshot 2023-01-30 at 11 58 41 AM" src="https://user-images.githubusercontent.com/5799712/215556746-a71032f6-408c-4260-b660-2721f537aab0.png">

Reply:
One way would be to code the path to the adding-users.json in the script instead of passing in a file as an argument.

Reply:
I removed the default path within the script, and now it must be specified in the command arguments. However adding any additional command line arguments such as the optional host, port, and database name, will cause an error to be given as it appears that it tries to load a file matching the next arguments given

Reply:
The node script no longer produces an error.